frameworks/hdfs/tests/test_sanity.py

```diff
@@ -205,9 +205,17 @@ def test_kill_all_datanodes():
 @pytest.mark.sanity
 @pytest.mark.recovery
 def test_permanently_replace_namenodes():
-    replace_name_node(0)
-    replace_name_node(1)
-    replace_name_node(0)
+    replace_node(0, "name")
+    replace_node(1, "name")
+    replace_node(0, "name")
+
+
+@pytest.mark.sanity
+@pytest.mark.recovery
+def test_permanently_replace_journalnodes():
+    replace_node(0, "journal")
+    replace_node(1, "journal")
+    replace_node(2, "journal")
```

Comment:
Let's replace `0` again as in the `name-node` test.

Reply:
This is going to fail pretty frequently until we get the readiness check under control.